File: Counter/src/main/java/com/kchmielewski/java/spring/core/Application.java
package com.kchmielewski.java.spring.core;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.Import;

/**
 * Application entry point.
 */
@SpringBootApplication
@Import(Config.class)
public class Application {
    public static void main(String[] args) {
        SpringApplication.run(Application.class);
    }
}

File: Counter/src/main/kotlin/com/kchmielewski/kotlin/spring/core/Application.kt
package com.kchmielewski.kotlin.spring.core

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.context.annotation.Import

@SpringBootApplication
@Import(Config::class)
open class Application

/**
 * Application entry point.
 */
fun main(args: Array<String>) {
    SpringApplication.run(Application::class.java, *args)
}
